__tests__/email.js

```diff
@@ -0,0 +1,13 @@
+import { reformatEmail } from "../utils/email.js";
```

Comment:
We can omit the `.js` ending here

```suggestion
import { reformatEmail } from "../utils/email";
```